include/zephyr/drivers/display.h

```diff
@@ -89,6 +89,10 @@ enum display_screen_info {
 	 * Screen has x alignment constrained to width.
 	 */
 	SCREEN_INFO_X_ALIGNMENT_WIDTH	= BIT(4),
+	/**
+	 * Screen will only present updated pixels after being told so
```

Comment:
Maybe expand this here, to something like so:

```
Screen will not show updated framebuffer to user until @ref display_show is called
```

Reply:
Fixed